extensions/ql-vscode/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## [UNRELEASED]
+## 1.8.4 - 3 May 2023
 
+- Avoid repeated error messages when unable to monitor a MRVA query. [#2396](https://github.com/github/vscode-codeql/pull/2396)
+- Fix MRVA not displaying the `#select` results set correctly when the query metadata does not exactly match results sets. [#2395](https://github.com/github/vscode-codeql/pull/2395)
+- Adds `codeQL.contextualQueries.disableCache` setting to help library authours to run and debug the find references and find dependencies contetextual queries without relying on the implicit cache. [#2385](https://github.com/github/vscode-codeql/pull/2385)
```

Comment:
I think this is internal enough that we don't need to include it in the changelog 🤔 

(But if you'd prefer to keep it, here are some typo fixes 😸 )
```suggestion
- Add `codeQL.contextualQueries.disableCache` setting to help library authors run and debug the "Find references" and "Find dependencies" contextual queries without relying on the implicit cache. [#2385](https://github.com/github/vscode-codeql/pull/2385)
```

Reply:
I wasn't sure whether to include this or not, so erred on the side of including it. But perhaps we shouldn't bother. It's not included in `package.json` so it isn't advertised to users. I'll remove this line.